Govspeak will not build

# Steps to reproduce

Join our plucky cast as they attempt to run:
`make govspeak`

## Stack trace

And encounter the mysterious case of the missing Argument 

```
/home/huw/govuk/govuk-docker/exe/govuk-docker run govspeak-lite rbenv install -s || (/home/huw/govuk/govuk-docker/exe/govuk-docker build --no-cache govspeak-lite; /home/huw/govuk/govuk-docker/exe/govuk-docker run govspeak-lite rbenv install -s)
docker-compose -f [...] run govspeak-lite rbenv install -s
/home/huw/govuk/govuk-docker/exe/govuk-docker run govspeak-lite sh -c 'gem install --conservative --no-document bundler -v $(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'
docker-compose -f [...] run govspeak-lite sh -c gem install --conservative --no-document bundler -v $(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)
grep: Gemfile.lock: No such file or directory
ERROR:  While executing gem ... (OptionParser::MissingArgument)
    missing argument: -v
make: *** [Makefile:41: bundle-govspeak] Error 1
```
##

Seems related to:
https://github.com/alphagov/govuk-docker/issues/389

Implies https://github.com/alphagov/govuk-docker/pull/396
May not be a complete fix.

--- a/Makefile
+++ b/Makefile
@@ -38,7 +38,7 @@ all-apps: $(APPS)
 bundle-%: clone-%
 	$(GOVUK_DOCKER) build $*-lite
 	$(GOVUK_DOCKER) run $*-lite rbenv install -s || ($(GOVUK_DOCKER) build --no-cache $*-lite; $(GOVUK_DOCKER) run $*-lite rbenv install -s)
-	if [ -f Gemfile.lock ]; then $(GOVUK_DOCKER) run $*-lite sh -c 'gem install --conservative --no-document bundler -v $$(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'; fi
+	if [ -f "${GOVUK_ROOT_DIR}/$*/Gemfile.lock" ]; then $(GOVUK_DOCKER) run $*-lite sh -c 'gem install --conservative --no-document bundler -v $$(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'; fi
 	$(GOVUK_DOCKER) run $*-lite bundle
 
 clone-%:

